### ISSUE TITLE
[Feature request] DAITA only on WIFI

### ISSUE THREAD

**Issue by secretmango** [mullvad/mullvadvpn-app#7366]
### I have checked if others have suggested this already

- [X] I have checked this issue tracker to see if others have reported similar issues.

### Feature description

DAITA sounds like a pretty essential technology. I have used it and in WIFI can totally live with the downsides.

But on cell data, I am pretty sure it was the cause of my extreme data usage, 6GB in a few hours. 

So I, and everyone else on a normal cell data plan, need to turn it off.

Instead I would like a feature where the VPN app detects the currently used network (metered state) and disables DAITA on metered networks.

This should work on Android, Linux and more.

### Alternative solutions

Disabling it entirely.

### Type of feature

- [X] Better privacy/anonymity
- [ ] Better at circumventing censorship
- [X] Easier to use
- [ ] Other

### Operating System

- [X] Android
- [ ] iOS
- [ ] Windows
- [ ] macOS
- [X] Linux

**Comment by kolon0** [comment]
As a Turkish citizen, I believe this feature is necessary due to the very limited (and expensive) nature of our mobile internet. While I am willing to tolerate the slowness that DAITA introduces when using Wi-Fi, it is impossible for me to use DAITA with mobile data. It consumes too much data.

**Comment by Serock3** [comment]
Thanks for reporting this. I can see that the suggestion has gotten some traction, so it's definitely something we want to consider. We have previously discussed the possibility of allowing settings to be configured on a per-network-basis, but it's nothing that we are working on at the moment. I'll bring this up with the team.

**Comment by boredsquirrel** [comment]
Yes please consider this! Alternatively people currently need unlimited cell data, which costs 45€ per month or more... so automatic switching is pretty crucial

**Comment by Rawa** [comment]
Closing as duplicate of https://github.com/mullvad/mullvadvpn-app/discussions/8931, if you want this feel free to upvote that discussion.
